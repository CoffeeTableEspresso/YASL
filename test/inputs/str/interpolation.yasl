let $x = 10
let $y = 12
echo "$x is #{$x}, #{$y}."
echo "$x is #{$x        }, #{$y}"
echo "$x is #{$x  }#{$y}  "
