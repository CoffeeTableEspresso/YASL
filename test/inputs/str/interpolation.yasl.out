$x is 10, 12.
$x is 10, 12
$x is 1012  
